app/src/main/java/com/mycompany/traveljournal/models/Post.java

```diff
@@ -120,12 +120,12 @@ public String toString() {
         return output;
     }
 
-    public boolean equals (Object o) {
+    /*public boolean equals (Object o) {
```

Comment:
This will cause bus in my map animation. I was relying on this method for sorting posts wrt their timestamp.